lldb/test/API/lang/cpp/namespace/main.cpp

```diff
@@ -56,10 +56,10 @@ namespace Foo = A::B;   // namespace alias
 
 using Foo::myfunc;      // using declaration
 
-using namespace Foo;    // using directive
+//removing namespace foo; for quality naming 
```

Comment:
The `using` directive was used here to make sure LLDB does the right thing when doing qualified lookup in the presence of various `DW_TAG_imported_module` and name shadowing. I don't think we should change this because it risks subtly changing the test coverage.

Reply:
Hi @jeevanghimire, please note that this is a test, so we must ensure that LLDB does the right thing regardless of the input it receives, even if said input is not considered best practices. The test was likely created to either ensure LLDB works in the case, or because it didn't use to work and  someone created a test at the same time the fix was done.

 As such, I don't think we should change this.